### DISCUSSION TITLE
Ghidra 10.4 - Invalid PDB Path / GUID / UID value after initializing symbol path and selecting Microsoft symbol URL

### DISCUSSION THREAD

**Discussion by jonbfid** [NationalSecurityAgency/ghidra#5995] [Q&A]
<img width="653" alt="Configure_Symbol_Server_Search_and_Load_PDB_for_tiny_exe_and_Ghidra__tiny_reversing" src="https://github.com/NationalSecurityAgency/ghidra/assets/111766005/8c9bb54f-4f5e-427b-a911-8ab4f2b7b1ab">
<img width="660" alt="Bad_PDB_Values_and_Load_PDB_for_tiny_exe_and_Ghidra__tiny_reversing" src="https://github.com/NationalSecurityAgency/ghidra/assets/111766005/d8d4c832-a4f8-45ed-856c-bab11a42f804">

I'm following the guide here https://www.tripwire.com/state-of-security/ghidra-101-loading-windows-symbols-pdb-files-in-ghidra-10-x but encountering an error.  This happens for any selected URL, and I am getting the green check before clicking OK.  I see that things have changed on the Microsoft side, is that impacting Ghidra?

**Comment by dev747368** [comment]
Thanks for the screenshot, that helps narrow things down.

The main thing I'm taking from it is that your "PDB Name" and "PDB unique ID" fields seem to be blank.  Those values are extracted from your binary and used when retrieving the pdb from the symbol server (in your case, the public microsoft symbol server, which probably isn't the problem here).

If you look at the screenshots of the guide you mentioned, they show that their binary (dns.exe) has a pdb name of "dns.pdb" and a unique uuid.

Ghidra extracts those values from the header of your PE binary during import, and also places them in the binary's properties screen (Help | About yourbinaryname) as "PDB Age", "PDB File", "PDB GUID".  If those properties are not present, then your binary doesn't have the necessary  information embedded in it to link it to a pdb file.  (or ghidra had some kind of error when during import when trying to parse those values)

You might want to try importing something that is a known-good PE file like c:\windows\system32\cmd.exe and if you can get PDB loading to work with that file.

**Comment by jonbfid** [comment]
Thank you, I figured it was something like that.  I'm using files from here http://www.phreedom.org/research/tinype/, so definitely not normal.  Appreciate the help.
